hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/query/QueryContextImpl.java

```diff
@@ -130,4 +132,10 @@ public ExchangeRateInfo exchangeRateInfo() {
         }
         return exchangeRateInfo;
     }
+
+    @NonNull
+    @Override
+    public FeeCalculator feeCalculator(@NonNull SubType subType) {
+        return null;
```

Comment:
Is there is an issue in place to fill this in?

Reply:
I think Michal or Richard will it in. I can definitely can create one

Reply:
ticket created https://github.com/hashgraph/hedera-services/issues/8543